integration/tests/bft/passthrough/suite.go

```diff
@@ -5,6 +5,9 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/onflow/flow-go/module/irrecoverable"
+	"github.com/onflow/flow-go/network/codec/cbor"
```

Comment:
why did these move up here? Seems like something's not quite right with the `goimports` config in your editor

Reply:
https://github.com/onflow/flow-go/pull/2599/commits/b4c43a4bf6190a2c77de8913a50dcf6345dd9b03